MaraBot/Core/PresetValidation.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+
+namespace MaraBot.Core
+{
+    /// <summary>
+    /// Holds validation logic for presets.
+    /// </summary>
+    public static class PresetValidation
+    {
+        /// <summary>
+        /// Randomizer version the validation is written for.
+        /// </summary>
+        public static readonly string VERSION = "1.22";
```

Comment:
We should use `const` here instead of `static readonly` as const is resolved at compilation. For const strings, this will be more efficient.

Also, could we rename this to `kVersion` to follow the same code convention as other constants.

```suggestion
        public const string kVersion = "1.22";
```